solvcon/tests/test_march.py

```diff
@@ -15,7 +15,7 @@
 class TestCreation(TestCase):
 
     def test_import(self):
-        from solvcon.march import gas
+        from libmarch import gas
```

Comment:
These tests were supposed to explicit test for `solvcon.march`.  Now for `libmarch`.